client/src/components/Message/Message.stories.js

```diff
@@ -0,0 +1,122 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import Message from "./Message";
+import defaultPic from "assets/images/profile-picture-1.png";
+import defaultPic2 from "assets/images/profile-picture-2.png";
+import defaultPic3 from "assets/images/profile-picture-3.png";
+
+const conversation1 = {
```

Comment:
All this data is great! But could we potentially move it to a separate file?

Reply:
Yup! Find it in InboxMessagePreviewData.js